### ISSUE TITLE
Post-launch analysis of /firefox/ page performance

### ISSUE THREAD

**Issue by ejregithub** [mozilla/bedrock#8005]
## Description

We launched the new /firefox/ page a week or so prior to Skyline and need to understand the impact of the changes, moving from a single product (Firefox Browser) to a multiple product marketing page with the suite of Firefox offerings.

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] Analysis is complete and delivered

#### Tasks
- [ ] Perform analysis and deliver

**Comment by sghoseWI** [comment]
Relevant Datasets: 

https://docs.google.com/spreadsheets/d/1NJTIjXEI10wqx_knz_NwhqliDxO5s1zHdrszd6ThtfM/edit#gid=500144031

https://docs.google.com/spreadsheets/d/1ly0WjhFTyEKtDp1aU4YUT5JLIImRHvrv_twkfWZu1Es/edit#gid=0

**Comment by sghoseWI** [comment]
Hi All,

As we know, on October 14th we launched the redesigned /firefox page.  Because we are now promoting our whole suite of products on the page, as opposed to primarily promoting the browser like we used to, we expected to see some moderate level of decline in session-to-download conversion volume and conversion rate on the page. Our hope was that this moderate decline would be sustainable and worthwhile in order to increase overall engagement across our product offerings.  

George and I looked into diagnosing the cause of some surprising results showing that performance on the page in Germany and France was particularly low (beyond what we would have expected) — when compared to performance in the English language Tier 1 countries (Data here: https://docs.google.com/spreadsheets/d/1NJTIjXEI10wqx_knz_NwhqliDxO5s1zHdrszd6ThtfM/edit#gid=500144031).   While overall page performance of /firefox has remained quite stable in the comparable period post-implementation of the new page vs. pre-implementation, DE and FR have been lagging behind.  

Specifically, session volume dropped 6.1% points, download volume dropped 21.8% points, and download conversion rates dropped by 8.2% points in DE in the comparable period.  In FR, session volume dropped 14.9% points, download volume dropped 31.9% points, and download conversion rates dropped by 8.7% points.  In contrast to these results, in the United States, session volume actually increased by 17.5% points, download volume increased by 19.4% points, and average download conversion rates increased by 1.2% points.

We looked into some potential causes for the relative decline in DE and FR.  By segmenting the traffic by medium, we found that organic traffic went up quite a bit in the better performing Tier 1 countries on /firefox (up 8% points in US and up 13% points in CA) while it went down 5% points in DE and 4% points in FR.

In the US, the increase in organic traffic, which has the highest relative conversion rate among traffic mediums, is bringing up the country’s overall download volume and overall download conversion rate.  Referral traffic converts at a lower rate in the US than organic, but it is such a small share of overall traffic in the country that aggregate performance is not hurt so much.

In DE, on the other hand, relative organic traffic (which also has the highest conversion rate among traffic medium) is decreasing, thereby bringing down DE’s overall conversion rates.  Referral traffic in DE now has a bigger share of overall traffic — thereby bringing down the country’s overall page performance.   A similar pattern is emerging in France as well. 

We do not yet know why organic traffic in the english-speaking Tier 1 countries is climbing relative to referral traffic — while the opposite seems to be happening in France and Germany.  Raphael looked into this as well and found that it seems that the session and download volume drop on /firefox in FR + DE is more than made up for by improved performance on /new in those countries. This insight eases some of the region specific concerns we had re: FR + DE.  

As a final check, I built a time-series model that was trained on global download data over the last two years on /firefox and forecasted out to current day.  We can see in the chart below that global performance is actually slightly out-performing forecasted expectations thus far: 

clip_image001.png

Since overall global performance on the page is steady, we may want to consider experimenting with different copy/design variations in DE + FR specifically to try to address the regional gaps in performance we’re seeing so far.  That said, the main takeaway here is that overall performance on /firefox is stable and regional performance lags are not overly concerning at this point.   

Best,

Saptarshi
